### DISCUSSION TITLE
Does Chakra have a Multiselect component?

### DISCUSSION THREAD

**Discussion by davesag** [chakra-ui/chakra-ui#3479] [Q&A]
I'm looking into replacing Material UI with a lighter, cleaner, UI Component library.  The primary motivation for this is the lack of a nice multiselect component in Material UI (they have one but it's ugly and hand to use)

Chakra looks amazing and a good fit for us, but I can't find any built-in multi-select component. I did find https://github.com/koolamusic/chakra-ui-autocomplete but it's WIP according to the readme, and we need something more mature than that.

Maybe it's just that I am looking for 'multiselect' but there is another way to achieve the same result.

**Comment by with-heart** [comment]
Unfortunately we don't have a multiselect component. This is something we plan to support, but there isn't any work being done on it currently.

**Reply by with-heart** [reply]
See #1293 for previous discussion

**Reply by davesag** [reply]
> Unfortunately we don't have a multiselect component. This is something we plan to support, but there isn't any work being done on it currently.

Okay thanks — we'll keep looking

**Comment by csandman** [comment]
There is still no official one yet, but I have put a lot of effort into making a wrapper for [`react-select`](https://react-select.com/home) to make it look like a chakra component, with a full styling system using chakra's style props.  This has the benefit of the maturity of `react-select` while still looking like a chakra component.

Check it out here: https://github.com/csandman/chakra-react-select

![image](https://user-images.githubusercontent.com/9214195/149229446-42058ced-228c-46d4-905c-1fd6ca831e96.png)
